operations/mimir-mixin/dashboards/reads.libsonnet

```diff
@@ -212,33 +212,56 @@ local filename = 'mimir-reads.json';
         $.panel(title) +
         $.queryPanel(
           [
-            'kube_horizontalpodautoscaler_spec_min_replicas{%s, horizontalpodautoscaler="%s"}' % [$.namespaceMatcher(), $._config.autoscaling.querier.hpa_name],
-            'kube_horizontalpodautoscaler_spec_max_replicas{%s, horizontalpodautoscaler="%s"}' % [$.namespaceMatcher(), $._config.autoscaling.querier.hpa_name],
-            'kube_horizontalpodautoscaler_status_current_replicas{%s, horizontalpodautoscaler="%s"}' % [$.namespaceMatcher(), $._config.autoscaling.querier.hpa_name],
+            'sum(kube_horizontalpodautoscaler_spec_min_replicas{%s, horizontalpodautoscaler=~"%s"})' % [$.namespaceMatcher(), $._config.autoscaling.querier.hpa_name],
+            'sum(kube_horizontalpodautoscaler_spec_max_replicas{%s, horizontalpodautoscaler=~"%s"})' % [$.namespaceMatcher(), $._config.autoscaling.querier.hpa_name],
```

Comment:
It looks weird summing min/max. I understand what you're doing here, but the original idea of displaying min/max was that the current number of replicas will be between that band. That's no more true. What if we do `min(kube_horizontalpodautoscaler_spec_min_replicas)` and `max(kube_horizontalpodautoscaler_spec_max_replicas)` instead? Would be slightly more accurate.

If you accept this suggestion, then shouldn't be a stack anymore.

Reply:
One of the lines of "current" could be under "min", but the topmost would be between min and max.

I think that having `min(min)` would be also misleading, as that would make that number show half of the real min.

Maybe I should just add another row of panels for `querier-burst` and `querier-burst-backup` and make that feature explicitly enabled?

Reply:
Discussed offline. I'm up to remove min and keep max as `max(max)`.

Reply:
I've left each one of the max values, as they might be different, and doing the max(max) could hide some of the autoscaler being at max value.